### PR TITLE
Implement adding NHID in `PatientEditScreen` - 1

### PR DESCRIPTION
**Pull request by janhavisinghh**
https://app.shortcut.com/simpledotorg/story/6390/implement-adding-nhid-in-patienteditscreen

### CODE REVIEW

File: app/src/main/java/org/simple/clinic/editpatient/EditPatientEvent.kt [hunk 1]
```diff
@@ -72,3 +72,5 @@ data class AlternativeIdChanged(val alternativeId: String) : EditPatientEvent()
 data class InputFieldsLoaded(val inputFields: InputFields) : EditPatientEvent()
 
 data class ColonyOrVillagesFetched(val colonyOrVillages: List<String>) : EditPatientEvent()
+
+object AddNHIDButtonClicked : EditPatientEvent()
```
Comment (Honey14) [inline app/src/main/java/org/simple/clinic/editpatient/EditPatientEvent.kt:76]: Add analytics, please

**Review by Honey14 — CHANGES_REQUESTED** [review]
Looks good overall, left one comment

File: app/src/main/java/org/simple/clinic/editpatient/EditPatientEvent.kt (hunk 1 above)
Comment (janhavisinghh) [reply app/src/main/java/org/simple/clinic/editpatient/EditPatientEvent.kt]: done

**Review by Honey14 — APPROVED** [review]

**Pull request merged**